app/models/profile.js

```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  bio: DS.attr('string'),
+  image: DS.attr('string'),
+  following: DS.attr('boolean'),
+  articles: DS.hasMany('article', { async: false, inverse: 'author' }),
```

Comment:
`articles` are a non-async relationship on `profile` and are loaded specifically when entering the `profile.index` route.